deployments/helm-chart/templates/controller-daemonset.yaml

```
@@ -128,4 +128,7 @@ spec:
           - -enable-prometheus-metrics={{ .Values.prometheus.create }}
           - -prometheus-metrics-listen-port={{ .Values.prometheus.port }}
           - -enable-custom-resources={{ .Values.controller.enableCustomResources }}
+{{- if .Values.controller.enableCustomResources  }}
```

Shouldn't this be `enableTLSPassthrough` ?
```suggestion
{{- if .Values.controller.enableTLSPassthrough  }}
```

 Same in the `deployment` template

---

nope. this is to prevent setting `-enable-tls-passthrough` to `true` if `enableTLSPassthrough` is `false`, since  `-enable-tls-passthrough` requires `-enable-custom-resources`. Otherwise, the IC will exit with an error.

---

Ok thanks, it wasn't clear to me that this logic was in the Helm chart instead of in the IC